### PR TITLE
Fixed traefik config generation

### PR DESCRIPTION
**Pull request by serge1peshcoff**
This should be the final step to allow us having the prod env by the way, didn't face any other issues

### CODE REVIEW

File: helper.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ init_boot ()
     touch "${DIR}"/secrets/acme.json # to avoid making it think it's a folder
     chmod 600 "${DIR}"/secrets/acme.json # Traefik doesn't let ACME challenge go through otherwise
```
Comment (linuxbandit) [inline helper.sh:37]: Can you add the `touch` I mentioned? so we avoid having a folder

**Review by linuxbandit — CHANGES_REQUESTED** [review]

File: helper.sh (hunk 1 above)
Comment (serge1peshcoff) [reply helper.sh:37]: is there a point in `touch` ing the file and then overriding it with the correct one?

Comment (serge1peshcoff) [reply helper.sh:37]: it's not supposed to be not presented

File: helper.sh [hunk 2]
```diff
@@ -35,6 +35,7 @@ init_boot ()
...
 
+    touch "${DIR}"/oms-global/docker/traefik/traefik.toml.template # to avoid making it think it's a folder
```
Comment (linuxbandit) [inline helper.sh]: I meant to touch the file not yet existing, i.e. `"${DIR}"/oms-global/docker/traefik/traefik.toml`

File: helper.sh (hunk 1 above)
Comment (linuxbandit) [reply helper.sh:37]: it is, because we're talking about different files apparently :) oops

File: helper.sh (hunk 2 above)
Comment (linuxbandit) [reply helper.sh]: that's where the misunderstanding came from :)

**Review by linuxbandit — CHANGES_REQUESTED** [review]

File: helper.sh (hunk 2 above)
Comment (serge1peshcoff) [reply helper.sh]: Whoops, my fault. I intended to touch the file and not the template, but just copied the wrong string lol.
Will fix today.

**Review by linuxbandit — APPROVED** [review]

**Pull request merged**